http/headers/Available-Dictionary.json

```diff
@@ -0,0 +1,39 @@
+{
+  "http": {
+    "headers": {
+      "Available-Dictionary": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/HTTP/Reference/Headers/Available-Dictionary",
```

Comment:
We're missing a `_compat: {}` wrapper here.

Reply:
Oops! And not just there. Added in https://github.com/mdn/browser-compat-data/pull/26257/commits/960443656a83c2d9d53906a3c33af46c11d59b5a

Reply:
Oh thanks. Now I'm getting more meaningful error messages - including about the missing spec.